lib/client-logger/src/index.ts

```diff
@@ -1,16 +1,25 @@
-const { console } = global;
-
-/* tslint:disable: no-console */
+import { window, LOGLEVEL } from 'global';
+import console from 'loglevel';
 
 export const logger = {
+  trace: (message: any, ...rest: any[]): void => console.trace(message, ...rest),
   debug: (message: any, ...rest: any[]): void => console.debug(message, ...rest),
-  log: (message: any, ...rest: any[]): void => console.log(message, ...rest),
+  log: (message: any, ...rest: any[]): void => window.console.log(message, ...rest),
```

Comment:
This is pretty surprising, what's up with this?

Reply:
`log` has no "level" to it. it's not part of `console` lib.

I could decide to make `log` equal to `info`?